notary-api/src/facade/ordersFacade.js

```diff
@@ -0,0 +1,11 @@
+import { BigNumber } from 'bignumber.js';
+import { dataOrderAt } from '../utils';
+
+const getNotarizationFee = async (orderAddress, notarizationPercentage) => {
+  const dataOrder = dataOrderAt(orderAddress);
+  const price = await dataOrder.price();
```

Comment:
I think that if we're using the web3 1.0 from now on we should get price doing: 
```javascript 
const price = await dataOrder.methods.price().call();
``` 
right?

Reply:
True